ray/operator/base/ray-operator-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ray-operator
+spec:
+  # Do not change: currently only 1 replica is supported
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ray-operator
+  template:
+    metadata:
+      labels:
+        app: ray-operator
+    spec:
+      restartPolicy: Always
+      serviceAccountName: ray-operator
+      containers:
+        - name: ray-operator
+          imagePullPolicy: Always
+          image: 'quay.io/thoth-station/ray-operator:v0.1.3'
```

Comment:
Ideally we'd have this image in the ODH org. Where's the dockerfile for this image defined?